src/adHtmlRender.js

```diff
@@ -0,0 +1,11 @@
+export function writeAdHtml(markup, insertHTML = document.body.insertAdjacentHTML.bind(document.body)) {
+    // remove <?xml> and <!doctype> tags
+    // https://github.com/prebid/prebid-universal-creative/issues/134
+    markup = markup.replace(/\<(\?xml|(\!DOCTYPE[^\>\[]+(\[[^\]]+)?))+[^>]+\>/g, '');
```

Comment:
`/g` -> `/gi` to use case-insensitive find & replace. Lowercase doctype is still valid and Amazon APS admarkup uses lowercase doctype. Without that, render of Amazon ads would fail due to the doctype still being present. Feel free to lift the test added over here https://github.com/prebid/prebid-universal-creative/pull/245/files then the other PR could be closed.